content/en/docs/instrumentation/js/exporters.md

```diff
@@ -7,10 +7,34 @@ In order to visualize and analyze your traces and metrics, you will need to expo
 
 Below you will find some introductions on how to setup backends and the matching exporters.
 
+- [OTLP endpoint or Collector](#otlp-endpoint-or-collector)
 - [Jaeger](#jaeger)
 - [Zipkin](#zipkin)
 - [Prometheus](#prometheus)
-- [OpenTelemetry Collector](#opentelemetry-collector)
+
+## OTLP endpoint or Collector
+
+To send trace data to a Collector you'll want to use an exporter package (e.g., `@opentelemetry/exporter-trace-otlp-http`):
+
+```shell
+npm install --save @opentelemetry/exporter-trace-otlp-http
```

Comment:
I've started an effort to standardize all of these to use the `console` block-code language specifier, and start the command with the `$` prompt.

This is an optional change, I'll be sweeping the docs to change it elsewhere, so it's up to you if you want to do it now.